### ISSUE TITLE
Sometimes breaks on parentheses in PATH

### ISSUE THREAD

**Issue by nosewings** [bucaran/darwin#18]
Minimal example:

```fish
set PATH "(foo)" $PATH
replay 'PATH=bar:$PATH'
```

The problem seems to be on [Line 33](https://github.com/jorgebucaran/replay.fish/blob/main/functions/replay.fish#L33=), which will yield the string `set PATH (foo) ...`.

This particular issue could be fixed by replacing Line 33 with the following:

```fish
string replace --all ')' '\)' (string replace --all '(' '\(' (string replace --all : " " "set $name $value"))
```

Not sure whether this is the best solution, though.

**Comment by jorgebucaran** [comment]
Why is your PATH set to `(foo)`?

**Comment by nosewings** [comment]
This came up under WSL, which will frequently have a path component like `/mnt/c/Program Files (x86)/...`.

**Comment by jorgebucaran** [comment]
I think I was able to fix this in bf47372. Please update and try again! 🙏

**Comment by nosewings** [comment]
Just noting that this did fix it. Awesome work!
